apps/api/src/campaign-application/campaign-application.service.ts

```diff
@@ -20,7 +22,7 @@ export class CampaignApplicationService {
   }
 
   findAll() {
-    return `This action returns all campaignApplication`
+    return this.prisma.campaignApplication.findMany()
```

Comment:
The controller would need to be updated to check for authenticated user and allow getting only for admin/operator roles.